itests/hcatalog-unit/src/test/java/org/apache/hive/hcatalog/listener/DummyRawStoreFailEvent.java

```diff
@@ -911,4 +912,11 @@ public void addPrimaryKeys(List<SQLPrimaryKey> pks)
   public void addForeignKeys(List<SQLForeignKey> fks)
       throws InvalidObjectException, MetaException {
   }
+
+  @Override
+  public Map<String, ColumnStatisticsObj> getAggrColStatsForTablePartitions(String dbName,
+      String tableName) throws MetaException, NoSuchObjectException {
+    // TODO Auto-generated method stub
```

Comment:
just finish this by doing this ? : return objectStore.getAggrColStatsForTablePartitions(dbName, tableName)